engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -1410,11 +1410,11 @@ namespace dmGameSystem
                 if (render_item->m_BoneIndex != dmRig::INVALID_BONE_INDEX)
                 {
                     dmRig::BonePose bone_pose = pose[render_item->m_BoneIndex];
-                    model_matrix = dmTransform::ToMatrix4(bone_pose.m_World) * dmTransform::ToMatrix4(render_item->m_Model->m_Local);
+                    model_matrix = dmTransform::ToMatrix4(bone_pose.m_World) * dmTransform::ToMatrix4(render_item->m_Model->m_World);
```

Comment:
This doesn't look right, multiplying two world matrix is not how you create a new world matrix?

Have you tested this code path with a skinned character?

Reply:
> Alright, I just committed a new push - [7fb10bc](https://github.com/defold/defold/commit/7fb10bc710cc4cefcdbfcdcfd1543dccd050e017)
> 
> And I test it in both editor and game by [gltf-animated+rigged+child.zip](https://github.com/user-attachments/files/17479019/gltf-animated%2Brigged%2Bchild.zip).

Yes, I tested.

Reply:
I will try to use the `local` for the bone node.
 But I think the `local` is a wrong parameter for child node.

Reply:
I just replace all `m_Local` to `m_World`, because it is no problem when I set the `m_Local` to node's world like before.